unicorn-play/src/main/scala/org/virtuslab/unicorn/SyncRep.scala

```diff
@@ -0,0 +1,27 @@
+package org.virtuslab.unicorn
+
+import slick.driver.JdbcDriver
+import slick.lifted.{ Rep, Query }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class SyncRep[T](rep: Rep[T], driver: JdbcDriver) {
```

Comment:
It is used in implicit conversion from Rep[T] in order to have .list and .run syntax which were in the previous Slick version. We need similar syntax because a lot of code currently is written that way.